src/apps/mpx.erl

```diff
@@ -79,8 +83,8 @@ handle_call({subscribe, #de_client{meta=Apps, data=AppData}=Client, App}, _, Sta
               gen_server:cast(?SERVER, {reply_to_client, NewClient, element(3, Tuple)});
               true -> pass
             end,
-            NewData = #{data => Data, encoder => Encoder},
-            {ok, Client#de_client{meta=[App|Apps], data=maps:put(App, NewData, AppData)}}
+            NewClientData = #{data => ClientData, encoder => Encoder},
+            {ok, Client#de_client{meta=[App|Apps], data=maps:put(App, NewClientData, AppData)}}
```

Comment:
Line 87 is too long:             {ok, Client#de_client{meta=[App|Apps], data=maps:put(App, NewClientData, AppData)}}.